app/server/api/auth_api.py

```diff
@@ -156,18 +156,14 @@ def post(self):
             if tfa_response_oject:
                 tfa_response_oject['auth_token'] = auth_token.decode()
 
+                db.session.commit()  # need this here to commit a created user to the db
+
                 return make_response(jsonify(tfa_response_oject)), 401
 
             # Update the last_seen TS for this user
             user.update_last_seen_ts()
 
-            response_object = {
-                'status': 'success',
-                'message': 'Successfully activated.',
-                'auth_token': auth_token.decode(),
-                'user_id': user.id,
-                'email': user.email,
-            }
+            response_object = create_user_response_object(user, auth_token, 'Successfully activated.')
```

Comment:
Not sure why this wasn't a proper `create_user_response_object`. User was able to login, but with errors, causing an auto-logout as soon as they clicked something